File: static/api1.R
library(httr)
library(jsonlite)

CREEDS_URL <- 'http://amp.pharm.mssm.edu/CREEDS/'
response <- GET(paste0(CREEDS_URL, 'search'), query=list(q ='STAT3'))
if (response$status_code == 200){
	response <- fromJSON(httr::content(response, 'text'))
	print(response)
}

File: creeds/static/api1.py
import json, requests
from pprint import pprint

CREEDS_URL = 'http://amp.pharm.mssm.edu/CREEDS/'
response = requests.get(CREEDS_URL + 'search', params={'q':'STAT3'})
if response.status_code == 200:
	pprint(response.json())
	json.dump(response.json(), open('api1_result.json', 'wb'), indent=4)
